base/int.jl

```diff
@@ -332,6 +332,21 @@ for to in BitInteger_types, from in (BitInteger_types..., Bool)
     end
 end
 
+"""
+    rem(x::Integer, T<:Integer)
```

Comment:
I think this notation is nonstandard with regards to the rest of the Base docstrings. Also note that `<:Integer` isn't entirely accurate, since you can't do something like `1 % BigInt`.